project/src/main/java/com/google/sps/utilities/KeywordHelper.java

```diff
@@ -0,0 +1,53 @@
+package com.google.sps.utilities;
+
+import com.google.cloud.language.v1.AnalyzeEntitiesRequest;
+import com.google.cloud.language.v1.AnalyzeEntitiesResponse;
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.Document.Type;
+import com.google.cloud.language.v1.EncodingType;
+import com.google.cloud.language.v1.Entity;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.sps.data.Keyword;
+import java.io.IOException;
+import java.util.ArrayList;
+
+/* Class that abstracts away getting keywords for specific content to enable testing. */
+public class KeywordHelper {
+  
+  /** Singleton instance of KeywordHelper. */
+  private static KeywordHelper instance;
+
+  /** Returns the singleton instance of KeywordHelper. */
+  public static KeywordHelper getInstance() {
+    if (instance == null) {
```

Comment:
nit: Look into some shorthand for this? I only know of such a thing in python, but I found a few links like this one that might have some one-liner ideas
https://stackoverflow.com/questions/32988485/java-different-approaches-to-handle-if-null-then-new/32988645

Reply:
Will keep in mind. Seems like writing it out long defines the purpose of the code clearly.